app/src/components/shared/AvatarSelectButton.tsx

```diff
@@ -0,0 +1,52 @@
+import * as ImagePicker from 'expo-image-picker';
+import React, { useState } from 'react';
+import { Platform, StyleSheet } from 'react-native';
+import { useTheme } from 'react-native-paper';
+
+import { AppButton } from './AppButton';
+
+interface AvatarButtonProps {
+  avatarUri: string;
+  onAvatarChange(uri: string): void;
+}
+
+export const AvatarSelectButton: React.FC<AvatarButtonProps> = (props) => {
+  const theme = useTheme();
+  const onAvatarButtonClick = async () => {
+    if (Platform.OS !== 'web') {
```

Comment:
Można się pozbyć zagnieżdżenia

```suggestion
    if (Platform.OS === 'web') {
      return;
    }
    
    // continue
```